From db0d59cd06f3ffd350379847c0885e1bfb85af0f Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 7 Mar 2015 11:34:42 +0100
Subject: [PATCH 2/2] build: add Libs.private field in libnl pkg-config file

In order to support static linking, the libnl pkg-config file should
indicate in its Libs.private field the libraries that libnl-3.0.a
requires. The LIBS variable contains the appropriate list of
libraries: -lm in all cases, and -lpthread when pthread support is
enabled. This allows to statically link applications against libnl
properly.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 libnl-3.0.pc.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libnl-3.0.pc.in b/libnl-3.0.pc.in
index b87e3dc..ddbc999 100644
--- a/libnl-3.0.pc.in
+++ b/libnl-3.0.pc.in
@@ -7,4 +7,5 @@ Name: libnl
 Description: Convenience library for netlink sockets
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -lnl-@MAJ_VERSION@
+Libs.private: @LIBS@
 Cflags: -I${includedir}/libnl@MAJ_VERSION@
-- 
2.1.0

